event_detection/bounce_detection.py

```diff
@@ -111,6 +111,8 @@ def detect_bounces(boxes, first_ball_data, prev_ball_data, frame_num, event_det_
     
     curr_ball_data = coord[i_value][:-1]
 
+  # TODO check for other types of bounces
```

Comment:
Intentional TODO commit? Fine if it is, just confirm it is for work on a different PR or whatever and not something you meant to do here

Reply:
Yeah that is something to do for another PR if we have time